maven-archetype-jakartaee9/src/main/resources/META-INF/maven/archetype-metadata.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
The Starter for Jakarta EE is under EPL-2.0. Any license headers must match this license.

Reply:
Clean license headers of all files.